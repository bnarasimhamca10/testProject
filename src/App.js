import React from "react";
import JSONServerData from "./components/JSONServerData";
import StyledCompButton from "./components/StyledCompButton";
import "./styles/styles.css";


const App = () => {

  return (
    <div>
      <h1> Normatl CSS</h1>
      <StyledCompButton />
      <JSONServerData />
    </div>
  );
};

export default React.memo(App);
