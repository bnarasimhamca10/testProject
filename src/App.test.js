import { create } from "react-test-renderer";
import App from './App';
import Enzyme from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';


// configure the adapter
Enzyme.configure( { adapter: new EnzymeAdapter() });
test( 'should render without errors', () => {
  let item = create(<App />);
  console.log(item.toJSON());
  expect(item.toJSON()).toMatchSnapshot();
} );