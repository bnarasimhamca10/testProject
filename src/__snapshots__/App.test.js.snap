// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without errors 1`] = `
<div>
  <h1>
     Normatl CSS
  </h1>
  <button
    className="sc-bdvvtL bzyOyq"
  >
    Styled Button A
  </button>
  <button
    className="sc-bdvvtL cKdtGz"
  >
    Styled Button B
  </button>
  <div
    className="red"
  />
</div>
`;
